psi4/src/psi4/dfocc/ccsd_triples.cc

```diff
@@ -2098,6 +2105,26 @@ void DFOCC::ccsd_canonic_triples_grad2() {
                 Miabd->contract(false, true, navirA * navirA, navirA, navirA, V, T, 0,
                                 (j * naoccA * navirA * navirA) + (k * navirA * navirA), 1.0, 1.0);
 
+
```

Comment:
You might want to consider something like:

```
double step_print = 5.0;
double next_print = step_print;

for (){
    double percent = (double)ind/Nijk
    if (percent >= next_print){
     next_print += step_print;
     print ...
}
```

Should be fewer lines and give a cleaner path to changing the percents later.

Reply:
good idea, I added I nicer version.